README.md

```diff
@@ -1 +1,4 @@
-# project_review
\ No newline at end of file
+# project_review
+
+#Created index.html file.
+#Created styles.css file.
```

Comment:
Please, your ReadMe.md  file should be very descriptive and should follow this template https://github.com/microverseinc/readme-template